src/dropdown/Dropdown.js

```diff
@@ -102,9 +102,18 @@ export default class Dropdown extends React.Component {
     return onChange(value);
   }
 
+  onInputKeyDown = (e) => {
```

Comment:
discussed offline: there cases where it does make sense (or simply where it may happen) to have a dropdown with no options